<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1149</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">display: flex;
  height: 92vh;
  /* flex-direction: row; */
  justify-content: center;
  align-items: center;
  color: rgb(0, 0, 108);
  background-color: #1b142d;
  /* margin-top: -80px; */
  font-family: cairo;
	border-radius:5%;

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>10</y>
      <width>261</width>
      <height>511</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">display: flex;
  height: 92vh;
  /* flex-direction: row; */
  justify-content: center;
  align-items: center;
  color: rgb(0, 0, 108);
  background-color: #1b142d;
  /* margin-top: -80px; */
  font-family: cairo;
	border-radius:5%;
</string>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="documentMode">
     <bool>true</bool>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>filters</string>
     </attribute>
     <widget class="QSpinBox" name="add_s">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>60</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">	border-radius:5%;
	border:3px dashed #5ac1ee;
	background-color:transparent;
	color:#5ac1ee;</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="add_g">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>60</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">	border-radius:5%;
	border:3px dashed #5ac1ee;
	background-color:transparent;
	color:#5ac1ee;</string>
      </property>
      <property name="maximum">
       <double>1.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="add_u">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>60</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">	border-radius:5%;
	border:3px dashed #5ac1ee;
	background-color:transparent;
	color:#5ac1ee;</string>
      </property>
      <property name="maximum">
       <double>1.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
     </widget>
     <widget class="QSpinBox" name="guasian">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>100</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">	border-radius:5%;
	border:3px dashed #5ac1ee;
	background-color:transparent;
	color:#5ac1ee;</string>
      </property>
      <property name="minimum">
       <number>3</number>
      </property>
      <property name="singleStep">
       <number>2</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="mean">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>100</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">	border-radius:5%;
	border:3px dashed #5ac1ee;
	background-color:transparent;
	color:#5ac1ee;</string>
      </property>
      <property name="minimum">
       <number>3</number>
      </property>
      <property name="singleStep">
       <number>2</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="median">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>100</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">	border-radius:5%;
	border:3px dashed #5ac1ee;
	background-color:transparent;
	color:#5ac1ee;</string>
      </property>
      <property name="minimum">
       <number>3</number>
      </property>
      <property name="singleStep">
       <number>2</number>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border-radius:5%;
	border:3px dashed #ffffff;

}
QPushButton:enabled{
	background-color:transparent;
	color:#ffffff;
}
 </string>
      </property>
      <property name="text">
       <string>upload</string>
      </property>
     </widget>
     <widget class="QComboBox" name="noise">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>60</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">	border-radius:5%;
	border:3px dashed #5ac1ee;
	background-color:transparent;
	color:#5ac1ee;</string>
      </property>
      <item>
       <property name="text">
        <string>spi</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>uniform</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>guasian</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="low">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>100</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">	border-radius:5%;
	border:3px dashed #5ac1ee;
	background-color:transparent;
	color:#5ac1ee;</string>
      </property>
      <item>
       <property name="text">
        <string>mean</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>median</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>guasian</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="high">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>140</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">	border-radius:5%;
	border:3px dashed #5ac1ee;
	background-color:transparent;
	color:#5ac1ee;</string>
      </property>
      <item>
       <property name="text">
        <string>berwit</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>robert</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>sobel</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>hybrid</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_22">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>90</y>
        <width>80</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">	border-radius:5%;
	border:3px dashed #5ac1ee;
	background-color:transparent;
	color:#5ac1ee;</string>
      </property>
      <property name="text">
       <string>Apply</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_6">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>90</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">	border-radius:5%;
	border:3px dashed #5ac1ee;
	background-color:transparent;
	color:#5ac1ee;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_27">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>190</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>prewitt</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_28">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>190</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>sobel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_29">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>193</y>
        <width>211</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(170, 0, 0);
background-color: rgb(170, 0, 0);</string>
      </property>
      <property name="text">
       <string>Robert</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_20">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>80</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border-radius:5%;
	border:3px dashed #ffffff;

}
QPushButton:enabled{
	background-color:transparent;
	color:#ffffff;
}
 </string>
      </property>
      <property name="text">
       <string>upload1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_21">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>40</y>
        <width>80</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border-radius:5%;
	border:3px dashed #ffffff;

}
QPushButton:enabled{
	background-color:transparent;
	color:#ffffff;
}
 </string>
      </property>
      <property name="text">
       <string>upload2</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>histogram</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_9">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>80</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border-radius:5%;
	border:3px dashed #ffffff;

}
QPushButton:enabled{
	background-color:transparent;
	color:#ffffff;
}
 </string>
      </property>
      <property name="text">
       <string>upload</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="normalize">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>60</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">	border-radius:5%;
	border:3px dashed #5ac1ee;
	background-color:transparent;
	color:#5ac1ee;</string>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="manual">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>60</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">	border-radius:5%;
	border:3px dashed #5ac1ee;
	background-color:transparent;
	color:#5ac1ee;</string>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="local">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>60</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">	border-radius:5%;
	border:3px dashed #5ac1ee;
	background-color:transparent;
	color:#5ac1ee;</string>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_31">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>80</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">	border-radius:5%;
	border:3px dashed #5ac1ee;
	background-color:transparent;
	color:#5ac1ee;</string>
      </property>
      <property name="text">
       <string>Equallization</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_32">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>180</y>
        <width>80</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">	border-radius:5%;
	border:3px dashed #5ac1ee;
	background-color:transparent;
	color:#5ac1ee;</string>
      </property>
      <property name="text">
       <string>canny</string>
      </property>
     </widget>
     <widget class="QComboBox" name="options">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">	border-radius:5%;
	border:3px dashed #5ac1ee;
	background-color:transparent;
	color:#5ac1ee;</string>
      </property>
      <item>
       <property name="text">
        <string>normalize</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>local</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>manual</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>auto</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="Histogram">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">	border-radius:5%;
	border:3px dashed #5ac1ee;
	background-color:transparent;
	color:#5ac1ee;
</string>
      </property>
      <item>
       <property name="text">
        <string>Histogram</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Gray</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Colored</string>
       </property>
      </item>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>40</y>
      <width>281</width>
      <height>281</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:3px dashed #5ac1ee;
border-radius: 5%;
 border-style: dashed;

  background-color: transparent;

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>330</y>
      <width>211</width>
      <height>211</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border:3px dashed #5ac1ee;
border-radius: 5%;
 border-style: dashed;

  background-color: transparent;

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>40</y>
      <width>281</width>
      <height>281</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:3px dashed #5ac1ee;
border-radius: 5%;
 border-style: dashed;

  background-color: transparent;

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>330</y>
      <width>201</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:3px dashed #5ac1ee;
border-radius: 5%;
 border-style: dashed;

  background-color: transparent;

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>330</y>
      <width>211</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:3px dashed #5ac1ee;
border-radius: 5%;
 border-style: dashed;

  background-color: transparent;

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1149</width>
     <height>0</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
